test/functional/fixtures/using-shared-code/testcafe-fixtures/shared-code.test.js

```diff
@@ -2,10 +2,10 @@
 '@page ./index.html';
 
 '@require :testModule';
-'@require ./mixin.js';
+'@require ../mixins/mixin.js';
```

Comment:
No need to put mixins and helpers in the separate dir. Also they belong to testcafe tests, so they should be in `testcafe-fixtures` dir as `mixins.js` and `helpers.js` respectively.